### ISSUE TITLE
Add notification if something changes are detected in timetable for nearest N days

### ISSUE THREAD

**Issue by maxkoshevoi** [maxkoshevoi/NureTimetable#18]
If after update something changes in timetable, user will be notified

Proposed N = 7

**Comment by Felkree** [comment]
Of course, there should be a notification, and also so that when passing to the application after this notification, you can immediately click the button and update the schedule. (according to the type of plate "Recommended to update").

**Comment by maxkoshevoi** [comment]
@Felkree this notification can only be sent AFTER update (how else can I compare old and new timetable). So for now, you'll need to trigger update by yourself, and after update is finished, notification will appear

**Comment by Felkree** [comment]
@maxkoshevoi Ok. So theme with autoupdate + notification will be the best change.

**Comment by maxkoshevoi** [comment]
@Felkree yes, but notification by itself will be useful too. You won't need to check timetable for changes after update this way
